content/docs/command-reference/diff.md

```diff
@@ -42,6 +42,11 @@ for example when `dvc init` was used with the `--no-scm` option.
 > file comparison, refer to this
 > [comment](https://github.com/iterative/dvc/issues/770#issuecomment-512693256).
 
+> Also note that support for rename detection is currently limited to files
```

Comment:
let's move it right after the paragraph rename is mentioned first

and simplify a bit:

`Note, rename detects only files which ... ` and probably remove `If a file's ....` part - just to simplify again

Reply:
or we can put together an example and put that detailed explanation (`If a files's content ...` part) into that example

Reply:
Simplified this note and added an example